site/src/components/PortForwardButton/PortForwardButton.tsx

```diff
@@ -29,7 +29,10 @@ export const PortForwardButton: React.FC<React.PropsWithChildren<PortForwardButt
   const styles = useStyles()
   const [port, setPort] = useState("3000")
   const { location } = window
-  const urlExample = `${location.protocol}//${port}--${workspaceName}--${agentName}--${username}.${location.host}`
+  const urlExample =
+    process.env.CODER_ENABLE_WILDCARD_APPS === "true"
```

Comment:
❓: is this something we expect users to set and use? If so, should we add it to the docs?

Reply:
I like how we're using `true` here. Looks like that's the pattern we're using for other env vars based on me searching the docs

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/188688058-ce4e7ab6-8c28-4ae0-bb0f-84b165a0cd19.png">

Reply:
Yes, it is expected. We are going to write docs for the port forward feature for sure!

Reply:
Awesome! Will that be in this PR or a follow-up? And who is going to do that?

Reply:
Yes. It will be probably @BrunoQuaresma and @bpmct